src/XenCert/StorageHandler.py

```diff
@@ -2431,7 +2433,7 @@ def Create(self):
                 try:
                     device_config['SCSIid'] = scsiId
                     XenCertPrint("The SR create parameters are %s, %s" % (util.get_localhost_uuid(self.session), device_config))
-                    sr_ref = self.session.xenapi.SR.create(util.get_localhost_uuid(self.session), device_config, '0', 'XenCertTestSR', '', 'lvmohba', '',False, {})
+                    sr_ref = self.session.xenapi.SR.create(util.get_localhost_uuid(self.session), device_config, '0', 'XenCertTestSR', '', self.sr_type, '',False, {})
```

Comment:
One very long line... never mind.
IMO we should run autopep8 on this codebase on some point.